src/components/activityEventElems/ActivityElement/ActivityElement.tsx

```diff
@@ -77,9 +77,11 @@ export function ActivityEvent({
 }: {
   header: string | JSX.Element
   subject: string | JSX.Element | null
-  event: ActivityElementEvent
+  event: ActivityElementEvent | null | undefined
   extra?: string | JSX.Element | null
 }) {
+  if (!event) return null
```

Comment:
Is it possible to use undefined instead of null? We use undefined everywhere in the codebase already so should try to keep the same I think

Reply:
i don't think functional components can return undefined? not seeing where we currently do that, i see lots of `return null`